examples/foreign-toplevel.c

```diff
@@ -29,10 +29,11 @@
  */
 
 enum toplevel_state_field {
-	TOPLEVEL_STATE_MAXIMIZED = 1,
-	TOPLEVEL_STATE_MINIMIZED = 2,
-	TOPLEVEL_STATE_ACTIVATED = 4,
-	TOPLEVEL_STATE_INVALID = 8,
+	TOPLEVEL_STATE_MAXIMIZED  = 1,
+	TOPLEVEL_STATE_MINIMIZED  = 2,
+	TOPLEVEL_STATE_ACTIVATED  = 4,
+	TOPLEVEL_STATE_FULLSCREEN = 8,
+	TOPLEVEL_STATE_INVALID    = 16,
```

Comment:
Can we use bitshifts for this?

Reply:
Yes. Should I change it in the wlroots header, too?

Reply:
That would be better yeah